pandas/io/tests/parser/usecols.py

```diff
@@ -254,3 +254,67 @@ def test_usecols_with_parse_dates_and_usecol_names(self):
                            usecols=[3, 0, 2],
                            parse_dates=parse_dates)
         tm.assert_frame_equal(df, expected)
+
+    def test_usecols_with_unicode_strings(self):
+        # see gh-13219
+
+        s = u'''AAA,BBB,CCC,DDD
+        0.056674973,8,True,a
```

Comment:
Don't make the data string unicode (all other `parser` tests don't).  You are just testing that `read_csv` accepts `usecols` with `unicode` strings.  If you want to include a test where `unicode` data is mixed with `unicode` and non-`unicode` strings, write that as a separate test from this one.